doc/site/content/docs/guides/getting-started/getting-started-with-rmlui.md

```diff
@@ -110,19 +116,23 @@ RmlUi.SetMouseCursorAlias("ew-resize", 'uiresizeh')
 RmlUi.CreateContext("shared")
 ```
 
-Also, add this line to your `luaui/main.lua`:
+This setup script can then be included from your `luaui/main.lua` (main.lua is the entry point for the LuaUI environment)
 
 ```lua
 VFS.Include(LUAUI_DIRNAME .. "rml_setup.lua",  nil, VFS.ZIP) -- Runs the script
 ```
 
-What this does is create a unified context for all your documents and data models, which is currently the reccommended way to architect documents. If you have any custom font files, list them in `font_files`, otherwise leave it empty.
+What this does is create a unified context 'shared' for all your documents and data models, which is currently the recommended way to architect documents. If you have any custom font files, list them in `font_files`, otherwise leave it empty.
```

Comment:
This is not true, `[basecontent rml_setup](https://github.com/beyond-all-reason/RecoilEngine/pull/2516/files)` does not create a shared context.

Reply:
Sounds good to me!